src/renderer/css/global.css

```diff
@@ -32,33 +32,47 @@ html.fa-events-icons-loading .fa-fw::before {
 }
 
 .ui.menu .item:focus {
-    outline: none;
+	outline: none;
 }
 
 #content {
-    height: 100%;
+	height: 100%;
 }
 
 #task-progress {
-    position: fixed;
-    width: 100%;
-    height: fit-content;
-    bottom: 3px;
+	position: fixed;
+	width: 100%;
+	height: fit-content;
+	bottom: 3px;
 }
 
 #progress-label-right {
-    text-align: right;
-    position: absolute;
-    width: 100%;
-    font-size: 1em;
-    top: 100%;
-    right: auto;
-    left: 0;
-    bottom: auto;
-    color: rgba(0,0,0,.87);
-    font-weight: 700;
-    text-shadow: none;
-    margin-top: .2em;
-    -webkit-transition: color .4s ease;
-    transition: color .4s ease;
+	text-align: right;
+	position: absolute;
+	width: 100%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5444487)